tools/wptrunner/wptrunner/executors/executorwebdriver.py

```diff
@@ -943,6 +944,20 @@ def set_display_features(self, features):
     def clear_display_features(self):
         return self.webdriver.send_session_command("DELETE", "displayfeatures")
 
+class WebDriverGlobalPrivacyControlProtocolPart(GlobalPrivacyControlProtocolPart):
+    def setup(self):
+        self.webdriver = self.parent.webdriver
+
+    def set_global_privacy_control(self, gpc):
+        body = {
+            "gpc": gpc,
+        }
+        return self.webdriver.send_session_command("POST", "privacy", body)
```

Comment:
For bonus points you could put an API for this in `tools/webdriver/webdriver/client.py` and use that rather than directly calling the HTTP endpoints here.